priv/templates/extended_bin

```diff
@@ -562,21 +562,9 @@ RELX_CONFIG_PATH=$(check_replace_os_vars sys.config "$RELX_CONFIG_PATH")
 set +e
 TMP_NAME_ARG=$(awk 'function check_name(file)
 {
-    if (system("test -f "file)) {
-        print file" not found"
-        exit 3
-    }
-    if (system("test -r "file)) {
-        print file" not readable"
-        exit 3
-    }
     while ((getline line<file)>0) {
         if (line~/^-args_file +/) {
             gsub(/^-args_file +| *$/, "", line)
-            if (!(line~/^\//)) {
-                print "relative path "line" encountered in "file
```

Comment:
This is the actual change in this PR, right? It is to now allow relative paths in vm args?

Reply:
yes, besides this it also removes the need for the included file to already exist when the start script runs, an allowed use case is having the start script hooks generate these (i'm actually going this route with a [plugin i'm writing ](https://github.com/lrascao/rebar3_scuttler)